drivers/adc/ad7124/ad7124_regs.c

```diff
@@ -2,8 +2,8 @@
 
```

Comment:
The commit needs to be signed, so do `git commit -s` when commiting.
This will add text in the form of:
`Signed-off-by: Name Surname <name.surname@domain.com>`